vgen/src/test/java/io/gapi/vgen/testdata/go_library.baseline

```diff
@@ -610,6 +610,118 @@ func (iterator *StringIterator) Next() (string, error) {
     iterator.currentIndex++
     return result, nil
 }
+============== file: library_service/example_test.go ==============
+package library_service
+
+func ExampleNewClient() {
+    ctx := context.Background()
+    opts := []gax.ClientOption{ /* Optional client parameters. */ }
+    client, err := library_service.NewClient(ctx, opts...)
+}
+
+func ExampleCreateShelf() {
+    ctx := context.Background()
+    req := &google_example_library_v1.CreateShelfRequest{ /* Data... */ }
+    opts := []gax.CallOption{ /* Optional call parameters. */ }
+    var resp *google_example_library_v1.Shelf
+    resp, err = client.CreateShelf(ctx, req, opts...)
```

Comment:
Have you tried `go test`ing this? I'm not sure if these samples compile.
- You probably need to do something like `_, _ = resp, err` to avoid compilation error
- Where is `err` defined here?